src/views/Referrals/Referrals.js

```diff
@@ -83,12 +98,15 @@ export default function Referral(props) {
   const [selectedRound, setSelectedRound] = useState("latest");
   const [nextRewards, setNextRewards] = useState();
 
-  const switchView = () => {
-    setCurrentView(currentView === COMMISSIONS ? REBATES : COMMISSIONS);
+  const eth = getTokenInfo(infoTokens, ethers.constants.AddressZero);
+  const ethPrice = eth?.maxPrimaryPrice;
+
+  const switchView = (view) => {
+    setCurrentView(view);
```

Comment:
Currently, the hash in the URL doesn't change when you click the buttons.  It might be better to replace `setCurrentView` here with `location.hash = "#theHash"` and allowing the useEffect hook below to change the state